packages/animations/lib/src/open_container.dart

```diff
@@ -51,11 +51,13 @@ enum ContainerTransitionType {
 ///
 ///  * [Transitions with animated containers](https://material.io/design/motion/choreography.html#transformation)
 ///    in the Material spec.
-class OpenContainer extends StatefulWidget {
+class OpenContainer<T> extends StatefulWidget {
```

Comment:
This should be annotated with `@optionalTypeArgs` and then you can use the <dynamic> from other parts of the code base.

Reply:
I annotated the class with `@optionalTypeArgs` but still the IDE shows errors when the `OpenContainer` is not explicitly provided type arguments.

```
Missing type arguments for generic type 'OpenContainer<dynamic>'.
Try adding an explicit type like 'dynamic', or enable implicit-dynamic in your analysis options file.
```

Reply:
Odd. You should be able to fix that with:

```suggestion
class OpenContainer<T extends Object> extends StatefulWidget {
```